### DISCUSSION TITLE
Is there a way to get the OpenIdConnectOptions from RequestServices?

### DISCUSSION THREAD

**Discussion by paulomorgado** [dotnet/aspnetcore#28158] [Q&A]
I want to generate an anchor for `OpenIdConnectOptions.OpenIdConnectOptions`.

**Comment by Tratcher** [comment]
> `OpenIdConnectOptions.OpenIdConnectOptions`.

Typo?

Is this related to your other question https://github.com/dotnet/aspnetcore/discussions/28153?

You can get access to options by resolving `IOptions<OpenIdConnectOptions>`, but I'd be surprised if you really need them. You shouldn't be linking directly to any of the paths in those options.

**Reply by paulomorgado** [reply]
That gets an initialized instance of `OpenIdConnectOptions`.

One of the configurations is the remote sign out path:

[`RemoteSignOutPath`](Requests received on this path will cause the handler to invoke SignOut using the SignOutScheme.)
> Requests received on this path will cause the handler to invoke SignOut using the SignOutScheme.

**Comment by paulomorgado** [comment]
This is the way to do it: https://github.com/dotnet/aspnetcore/blob/master/src/Identity/UI/src/Areas/Identity/Pages/V4/Account/Logout.cshtml
